plugins/dell-kestrel/fu-dell-kestrel-ec-hid.c

```diff
@@ -11,22 +11,6 @@
 
 #include "fu-dell-kestrel-ec-hid.h"
 
-typedef struct __attribute__((packed)) { /* nocheck:blocked */
-	/* header */
-	guint8 cmd;
-	guint8 ext;
-	guint32 data_sz;
-
-	/* data */
-	struct FuHidv2FwupdateDevFwInfo {
-		guint8 subcmd;
-		guint8 dev_type;
-		guint8 dev_identifier;
-		guint32 fw_sz;
-	} __attribute__((packed)) dev_fw_info; /* nocheck:blocked */
-	guint8 *fw_data;
-} FuEcHidFwupPkg;
-
 typedef struct __attribute__((packed)) { /* nocheck:blocked */
```

Comment:
can we get rid of this one too please? I don't really like seeing `nocheck` in new plugins otherwise it gives other vendors a get-out-clause too.

Reply:
Fixed